docs/source/tools/navigator/index.rst

```diff
@@ -23,12 +23,12 @@ Installing and starting Navigator
 
 Navigator ships with the DAML SDK. To launch it:
 
-1. Start Navigator via a terminal window running :doc:`SDK Assistant </tools/assistant>` by typing ``da start``
+1. Start Navigator via a terminal window running :doc:`SDK Assistant </tools/assistant>` by typing ``daml start``
 
 2. The Navigator web-app is automatically started in your browser. If it fails to start,
    open a browser window and point it to the Navigator URL
 
-  When running ``da start`` you will see the Navigator URL. By default it will be `<http://localhost:7500/>`_. However, if port ``7500`` is taken, a different port will be picked.
+  When running ``daml start`` you will see the Navigator URL. By default it will be `<http://localhost:7500/>`_. However, if port ``7500`` is taken, a different port will be picked.
```

Comment:
We don’t pick a new port automatically in the new assistant atm.

Reply:
What is the new behaviour - it'll fail?

Reply:
Yeah, we should probably make sure that we at least get a reasonable error message in that case.

Reply:
Created the issue #764 to track this.